client/src/components/profile/components/stats.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import Stats from './stats';
+
+const props: { calendar: { [key: number]: number }; points: number } = {
+  calendar: {},
+  points: 0
+};
+
+describe('<Stats/>', () => {
+  it('calculates the correct longest streak', () => {
+    render(<Stats {...props} />);
+    expect(screen.getByTestId('longest-streak')).toHaveTextContent(
+      'profile.longest-streak'
+    );
+  });
+
+  it('calculates the correct current streak', () => {
```

Comment:
i have a feeling im missing something here: whta is the difference between 2 tests?

Reply:
Between this one and the "longest streak" test?

Reply:
ok, sorry, my bad - now i see: difference is `current streak` vs `longest streak`